keras/backend/tensorflow_backend.py

```diff
@@ -4496,6 +4531,7 @@ def ctc_decode(y_pred, input_length, greedy=True, beam_width=100,
                 Important: blank labels are returned as `-1`.
             Tensor `(top_paths, )` that contains
                 the log probability of each decoded sequence.
+    {{np_implementation}}
```

Comment:
CTC decode isn't finished yet. Let's add it to the docs only when finished.

Reply:
I thought beam search is not implemented yet, but maybe greedy_search is available in numpy_backend.  
Is this also confusing? If so, maybe add it when finished.